composer/core/precision.py

```diff
@@ -38,11 +38,12 @@ class Precision(StringEnum):
 
 
 @contextlib.contextmanager
-def get_precision_context(precision: Union[str, Precision]) -> Generator[None, None, None]:
+def get_precision_context(precision: Union[str, Precision], precision_config: Dict[str, Any]) -> Generator[None, None, None]:
```

Comment:
Could you default this to None please? I don't want to break stuff which uses this API outside composer.

Reply:
done!